src/data/equiv/algebra.lean

```diff
@@ -328,3 +327,73 @@ instance symm.is_ring_hom {e : α ≃r β} : is_ring_hom e.to_equiv.symm := hom
   e.symm.to_equiv x = e.to_equiv.symm x := rfl
 
 end ring_equiv
+
+-- We first define the type of automorphisms on groups, additive monoids,
+-- additive groups and rings using `mul_equiv`, `add_equiv` and `ring_equiv`.
+-- In each case this type is also a group.
+
+namespace group
+
+def aut (γ : Type) [group γ] := mul_equiv γ γ
+
+@[extensionality] lemma ext_mul_equiv (α β : Type) [group α] [group β] (f : mul_equiv α β)
```

Comment:
Possibly rename to `mul_equiv.ext`? And maybe don't put this one inside the `group` namespace.

Reply:
Should I put in a namespace like ```group_aut``` or just in no namespace?

Reply:
I think all of these ext lemmas should be of the form `mul_equiv.ext` (no enclosing namespace). The point is that `mul_equiv` and friends are just in the root namespace, and these lemmas should be attached directly to them.

Reply:
done